File: alphanumeric.js
function alphanumeric(string) {
    console.log(RegExp("^[a-zA-Z0-9]{1,}$").test(string));
}

alphanumeric("Mazinkaiser")
alphanumeric("hello world_")
alphanumeric("PassW0rd")
alphanumeric("_")

File: alphanumeric.py
import re


def alphanumeric(password):
    if len(password) > 0 and re.match("^[A-Za-z0-9]*$", password):
        return True
    else:
        return False


alphanumeric("hello world_")
alphanumeric("PassW0rd")
alphanumeric("     ")
